lib/jalaali/calendar.ex

```diff
@@ -281,10 +290,55 @@ defmodule Jalaali.Calendar do
       59
 
   """
-  @impl true
+  @spec day_of_year(year, month, day) :: 1..366
   def day_of_year(year, month, day)
       when is_integer(year) and is_integer(month) and is_integer(day) do
     first_day_of_year = Jalaali.jalaali_to_days(year, 01, 01)
     Jalaali.jalaali_to_days(year, month, day) - first_day_of_year + 1
   end
+
+  ###########
+  # Helpers #
+  ###########
+
+  @spec offset_to_string(number, number, any, any) :: binary
+  defp offset_to_string(utc, std, zone, format \\ :extended)
+  defp offset_to_string(0, 0, "Etc/UTC", _format), do: "Z"
+
+  defp offset_to_string(utc, std, _zone, format) do
+    total = utc + std
+    second = abs(total)
+    minute = second |> rem(3600) |> div(60)
+    hour = div(second, 3600)
+    format_offset(total, hour, minute, format)
+  end
+
+
+  defp format_offset(total, hour, minute, :extended) do
+    sign(total) <> zero_pad(hour, 2) <> ":" <> zero_pad(minute, 2)
+  end
+
+  defp format_offset(total, hour, minute, :basic) do
```

Comment:
Hi @alisinabh, 
I merge Helper module into `Jalaali.Calendar` and delete the file concerned, and I tested your last update on master, it worked.

I think this function  will never be used!!. Would you mind testing this function and informing me, am I right or not? Please!

I think it should be deleted

Reply:
You can comment this for now. I think it was to support the Date module but we can look into all errors in a different PR with a CI pipeline.

Reply:
I commented it for now, and I am done, if you have any ideas I can do this in PR.

I'm very sorry for all changes 

Thanks